handler.js

```diff
@@ -237,122 +236,103 @@ module.exports.build_done = (event, context, callback) => {
         target_url: 'https://' + region + '.console.aws.amazon.com/codebuild/home?region=' + region + '#/builds/' + event.build.id + '/view/new',
         context: githubContext,
         description: 'Build ' + buildStatus + '...'
-      }).catch(function(err){
-        console.log(err);
-        context.fail(data);
```

Comment:
where did this one go?

Reply:
I intentionally removed it as data is undefined.

Reply:
Should I put it back as `context.fail(err)`

Reply:
actually, as far as i can tell, in current versions of the aws sdk for javascript, the `context` object no longer has a `fail()` method. so probably needs to be removed throughout, in favor of handling with `callback(err)`...

Reply:
Putting back as `context.fail(err)`

Reply:
https://docs.aws.amazon.com/lambda/latest/dg/nodejs-handler.html#nodejs-handler-sync

Reply:
Ok, I think this is fixed.